pkg/network/node/networkpolicy.go

```diff
@@ -330,6 +351,8 @@ func (np *networkPolicyPlugin) syncFlows() {
 			npns.mustSync = false
 		}
 	}
+	otx.DeleteFlows("table=27, cookie=%d/-1", NetPolIsolationCookie)
+	otx.DeleteFlows("table=80, cookie=%d/-1", NetPolIsolationCookie)
```

Comment:
a comment might be good here. "Remove the temporary flows added by SetUpPod now that all flows have been updated for new pods"